mbuild/lattice.py

```diff
@@ -226,7 +228,17 @@ def _validate_lattice_spacing(self, lattice_spacing):
         self.lattice_spacing = lattice_spacing
 
     def _validate_angles(self, angles):
-        """Ensure that the angles between the lattice_vectors are correct"""
+        """Ensure that the angles between the lattice_vectors are correct
+
+        _validate_angles will ensure that the angles input has an appropriate
```

Comment:
```suggestion
        Ensures that the angles input has an appropriate
```
For these, including similar lines elsewhere, you don't need to repeat the name of the function